wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/ServerChecker.java

```diff
@@ -119,8 +119,7 @@ public void run() {
       if (isTimedOut()) {
         reportFuture.completeExceptionally(
             new InfrastructureException(
-                String.format(
-                    "Server '%s' in machine '%s' not available.", serverRef, machineName)));
+                String.format("Server '%s' in '%s' is not available.", serverRef, machineName)));
```

Comment:
Why not replacing "machine" with "container"? This change makes it hard to understand the message.